### ISSUE TITLE
"NameError: global name 'cmdline' is not defined" in pydot 1.4.0

### ISSUE THREAD

**Issue by projectbtle** [pydot/pydot#201]
Hello,

When generating a control flow graph via `Androguard` (which uses `pydot`), the execution terminates with an error in pydot.py:
```
Dump Landroid/support/v4/app/q; j ()Landroid/os/Parcelable; ... jpg ... Traceback (most recent call last):
  File "C:\Users\XXX\.windows-build-tools\python27\Scripts\androdd.py", line 175, in <module>
    args.jar, args.decompiler, args.format)
  File "C:\Users\XXX\.windows-build-tools\python27\Scripts\androdd.py", line 109, in export_apps_to_format
    method2format(filename + "." + form, form, None, buff)
  File "C:\Users\XXX\.windows-build-tools\python27\lib\site-packages\androguard\core\bytecode.py", line 393, in method2format
    getattr(g, "write_" + _format.lower())(output)
  File "C:\Users\XXX\.windows-build-tools\python27\lib\site-packages\pydot.py", line 1734, in new_method
    encoding=encoding)
  File "C:\Users\XXX\.windows-build-tools\python27\lib\site-packages\pydot.py", line 1817, in write
    s = self.create(prog, format, encoding=encoding)
  File "C:\Users\XXX\.windows-build-tools\python27\lib\site-packages\pydot.py", line 1936, in create
    cmdline=cmdline,
NameError: global name 'cmdline' is not defined
```

I am using pydot v1.4.0 on Windows. This particular error is not present in pydot v1.2.4.

**Comment by prmtl** [comment]
There was a problem with the code that prints debug info after fail of rendering process. Now debug info will be correctly printed so you will be able to see what was the problem with the call. Please report another issue if there will be other problems.

**Comment by prmtl** [comment]
Fixed in 1.4.1
